domains/dunkiegaming.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "dunkiegaming",
+    "email": "duncangabrielse9@gmail.com"
+  },
+  "record": {
+    "A": "86.93.219.240"
```

```suggestion
    "A": ["86.93.219.240"]
```
Should be in an array.